topydo/lib/Config.py

```diff
@@ -186,6 +193,49 @@ def hidden_tags(self):
         hidden_tags = self.cp.get('ls', 'hide_tags')
         return [] if hidden_tags == '' else hidden_tags.split(',')
 
+    def priority_colors(self):
+        """ Returns a dict with priorities as keys and color numbers as value. """
+        pri_colors_str = self.cp.get('colorscheme', 'priority_colors')
+
+        def _str_to_dict(p_string):
+            pri_colors_dict = dict()
+            for pri_color in pri_colors_str.split(','):
```

Comment:
> Instead, I would expect +blue to turn up... well... blue.

You're right. I'll try to implement it. I'm thinking about something like this:

```python
def get_project_color(self):
        if self.project_color == '':
            return ''
        else:
            return self._get_ansi(self.project_color, 'bold')
```

This way project_color wouldn't add any ansi to output, so `+project` should retain previously defined color, which should be priority_color or NEUTRAL_COLOR (if priority_color is undefined). I'll try it out and send patch in short time.

> Returning to your question for the priorities, that would mean that all todo items will appear neutrally colored when they upgrade topydo, since they don't have a configuration file where priority_colors is defined.